core/src/test/java/org/elasticsearch/search/aggregations/bucket/significant/SignificantLongTermsTests.java

```diff
@@ -35,22 +37,22 @@
 import java.util.Map;
 import java.util.Set;
 
-import static org.elasticsearch.search.aggregations.InternalAggregations.EMPTY;
-
 public class SignificantLongTermsTests extends InternalSignificantTermsTestCase {
 
     private SignificanceHeuristic significanceHeuristic;
+    private DocValueFormat format;
 
     @Before
     public void setUpSignificanceHeuristic() {
         significanceHeuristic = randomSignificanceHeuristic();
+        format = randomNumericDocValueFormat();
```

Comment:
I think that to make this work as part of AggregationsTests you need to rename this method to setUp and remove before annotation?